CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+- Add `Coinbase.configure` method to allow for configuration of the SDK and marked constructor as deprecated.
```

Comment:
we should add `## [0.5.0] - 2024-09-11` here right? unless you plan to in a separate PR

Reply:
added in #235